modulemd/tests/test-modulemd-component-rpm.c

```diff
@@ -418,7 +418,7 @@ component_rpm_test_parse_yaml (void)
   g_assert_cmpint (event.type, ==, YAML_SCALAR_EVENT);
   yaml_event_delete (&event);
 
-  r = modulemd_component_rpm_parse_yaml (&parser, "bar", TRUE, &error);
+  r = modulemd_component_rpm_parse_yaml (&parser, "bar", TRUE, FALSE, &error);
```

Comment:
Should a another C test be added using the same input file but with `only_packager = TRUE` to make sure it grumbles about the presence of `buildorder`? I do see you added a Python test for that, so it would be someone redundant.

Reply:
That's probably a good idea.